### PR TITLE
[AUD-1409] Add profile-page tracks and reposts tabs

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds tracks and reposts tabs, leveraging the Lineup component

### Dragons

There was a weird bug in `Artwork.tsx` where the dominant color values where not integers and instead something like 120.0000039, which caused a rendering error for the shadow component. Added a workaround using let syntax, curious if folks like that approach vs a purely functional one

### CODE REVIEW

File: src/screens/profile-screen/RepostsTab.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      <EmptyTab>
+        <Text>{"You haven't reposted anything yet"}</Text>
+      </EmptyTab>
+    )
+  }
+
+  return (
+    <Lineup
+      actions={feedActions}
+      lineup={lineup as any}
```
Comment (dylanjeffers) [inline src/screens/profile-screen/RepostsTab.tsx:43]: going to work on the typing here, current issue is
```
Property '_loading' is missing in type 'LineupStateTrack<{ id: number; }>' but required in type 'LineupItem'.ts(2322)
[types.ts(14, 3): ]()'_loading' is declared here.
```
any immediate thoughts on what to do here?

Comment (sliptype) [reply src/screens/profile-screen/RepostsTab.tsx:43]: I think we should remove `_loading` from `LineupItem` and do 

```ts
    if ((item as LoadingLineupItem)._loading) {
```

here: https://github.com/AudiusProject/audius-mobile-client/blob/master/src/components/lineup/Lineup.tsx#L194

File: src/screens/profile-screen/EmptyTab.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  emptyTabRoot: {
+    marginVertical: spacing(2),
+    marginHorizontal: spacing(3)
+  },
+  emptyTab: {
+    paddingVertical: spacing(8),
+    paddingHorizontal: spacing(6)
+  },
+  emptyTabContent: {
+    display: 'flex',
```
Comment (sliptype) [inline src/screens/profile-screen/EmptyTab.tsx:16]: Weird flex but ok

(not needed)

File: src/screens/profile-screen/RepostsTab.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  const handlePauseTrack = useCallback(() => {
+    dispatchWeb(feedActions.pause())
+  }, [dispatchWeb])
+
+  if (!profile) return null
+
+  if (profile.repost_count === 0) {
+    return (
+      <EmptyTab>
+        <Text>{"You haven't reposted anything yet"}</Text>
```
Comment (sliptype) [inline src/screens/profile-screen/RepostsTab.tsx:35]: Let's use a `messages` object

**Review by sliptype — APPROVED** [review]
Looking awesome 🔥

File: src/screens/profile-screen/RepostsTab.tsx (hunk 1 above)
Comment (dylanjeffers) [reply src/screens/profile-screen/RepostsTab.tsx:47]: mind double checking the route i took?

Comment (sliptype) [reply src/screens/profile-screen/RepostsTab.tsx:47]: I think that's fine, but `_isLoading` will never exist on a lineup item. I added that field as a flag for use inside the Lineup logic to display extra skeletons

But I can go back and clean that up when tackling the rest of the Lineup changes

Comment (dylanjeffers) [reply src/screens/profile-screen/RepostsTab.tsx:47]: ohh gotcha

**Pull request merged**